source/_integrations/velux.markdown

```diff
@@ -20,6 +20,8 @@ There is currently support for the following device types within Home Assistant:
 - Cover
 - Scene
 
+The Velux Active (KIX 300) set is not supported by this integration. To integrate Velux Active (KIX 300) with Home Assistant, you can use the [HomeKit Controller](https://github.com/home-assistant/home-assistant.io/blob/current/integrations/homekit_controller) integration and get full control over your windows, curtains, covers etc. and also the air quality sensor KLA 300. Add the Velux Active gateway using HomeKit pairing (with the pairing code on the sticker at the bottom of the Velux Active gateway) and the devices connected to the gateway - including sensors - will be automatically discovered and added to Home Assistant.
```

Comment:
Can we move this into its own section at the bottom of the page? Instead of adding it to the introduction?

```suggestion
The Velux Active (KIX 300) set is not supported by this integration. To integrate Velux Active (KIX 300) with Home Assistant, you can use the [HomeKit Controller](/integrations/homekit_controller) integration and get full control over your windows, curtains, covers, the air quality sensor KLA 300, etc.

Add the Velux Active gateway using HomeKit pairing (with the pairing code on the sticker at the bottom of the Velux Active gateway) and the devices connected to the gateway - including sensors - will be automatically discovered and added to Home Assistant.
```

Reply:
My comment was not fully resolved.